align/pnr/checker.py

```diff
@@ -28,12 +28,11 @@ def check_placement(placement_verilog_d, scale_factor):
         with types.set_context(constraints):
             constraints.append(
                 constraint.SetBoundingBox(
-                    instance=module['concrete_name'],
+                    instance='subcircuit',
```

Comment:
This doesn't have to change now but the word `instance` here bothered me before. This thing is not an instance of another block (like this construct is being used elsewhere.) It is the bounding box of the enclosing block. We should probably have two types of bounding boxes and use a different parameter name (say `template_name` and `instance_name`.)

Reply:
I renamed the constraint itself to AssignBboxVariables & change instance to say bbox_name.

This constraint was even confusing me since its an internal constraint that pretty much gets used to check the values returned by the primitive generator & Placer.